### PR TITLE
update alert rules page, remove manage tasks

### PR DESCRIPTION
**Pull request by lwandzura**
Closes https://github.com/influxdata/docs-v2/issues/3920

removed outdated screenshots
removed manage tasks references and updated to alert rules
documented alert rules page options
updated alert rule builder section
updated alert history section

### CODE REVIEW

File: content/chronograf/v1.9/guides/create-alert-rules.md [hunk 1]
```diff
@@ -22,141 +22,70 @@ Common alerting use cases that can be managed using Chronograf include:
...
 
-Let it run for a few seconds or minutes before terminating it.
-On most systems, kill the script by using `Ctrl+C`.
+3. Select the time series data. 
+    - Choose the time series data you want the Chronograf alert rule to use.
+    - Navigate through databases, measurements, tags, and fields to select all relevant data.
 
-### Step 2: View the alerts
+4. Define the rule condition(s).
+    - Condition options are determined by the alert type.
```
Comment (sanderson) [inline content/chronograf/v1.9/guides/create-alert-rules.md:66]:
```suggestion
4. Define the rule conditions. Condition options are determined by the alert type.
```

File: content/chronograf/v1.9/guides/create-alert-rules.md [hunk 2]
```diff
@@ -22,141 +22,70 @@ Common alerting use cases that can be managed using Chronograf include:
...
-### Step 2: View the alerts
+4. Define the rule condition(s).
+    - Condition options are determined by the alert type.  
 
-Go to the Slack channel that you specified in the previous section.
-In this example, it's the `#chronocats` channel.
+5. Select and configure the alert handler.
+    - The alert handler determines where the system sends the alert (the event handler).
+    - Chronograf supports several event handlers and each handler has unique configurable options.
+    - Multiple alert handlers can be added to send alerts to multiple endpoints.
```
Comment (sanderson) [inline content/chronograf/v1.9/guides/create-alert-rules.md:71]:
```suggestion
5. Select and configure the alert handler.
    The alert handler determines where the system sends the alert (the event handler).
    Chronograf supports several event handlers and each handler has unique configurable options.
    Multiple alert handlers can be added to send alerts to multiple endpoints.
```

File: content/chronograf/v1.9/guides/create-alert-rules.md [hunk 3]
```diff
@@ -22,141 +22,70 @@ Common alerting use cases that can be managed using Chronograf include:
...
 
-Assuming the first step was successful, `#ohnos` should reveal at least two alert messages:
+6. Configure the alert message.
+    - The alert message is the text that accompanies an alert.
+    - Alert messages are templates that have access to alert data.
+    - Available templates appear below the message text field.
+    - As you type your alert message, clicking the data templates will insert them at the end of whatever text has been entered.
```
Comment (sanderson) [inline content/chronograf/v1.9/guides/create-alert-rules.md:77]:
```suggestion
6. Configure the alert message.
    The alert message is the text that accompanies an alert.
    Alert messages are templates that have access to alert data.
    Available templates appear below the message text field.
    As you type your alert message, clicking the data templates will insert them at the end of whatever text has been entered.
```

File: content/chronograf/v1.9/guides/create-alert-rules.md [hunk 4]
```diff
@@ -22,141 +22,70 @@ Common alerting use cases that can be managed using Chronograf include:
...
 
-![Chronograf alert history](/img/chronograf/1-6-alerts-history.png)
+To view a history of your alerts, click on
+**{{< icon "alert" "v2">}} Alerting** in the left navigation bar and select **Alert History**. 
+Do one of the following:
 
-That's it! You've successfully used Chronograf to configure an alert rule to monitor your idle CPU usage and send notifications to Slack.
+  - View a history of all triggered alerts.
+  - Filter alert history by type.  
+  - View alert history for a specified time range.
```
Comment (sanderson) [inline content/chronograf/v1.9/guides/create-alert-rules.md:91]: For the sake of thoroughness, let's add the following sections. I know they'll be short.

- Enable and disable an alert rule
- Delete an alert rule

**Review by sanderson — CHANGES_REQUESTED** [review]

**Review by kelseiv — APPROVED** [review]

**Pull request merged**